luigi/parameter.py

```diff
@@ -117,7 +117,7 @@ def run(self):
     _counter = 0  # non-atomically increasing counter used for ordering parameters.
 
     def __init__(self, default=_no_value, is_global=False, significant=True, description=None,
-                 config_path=None, positional=True, always_in_help=False):
+                 config_path=None, positional=True, always_in_help=False, batch_method=None):
```

Comment:
You really should add docs for this. Perhaps just linking to the `luigi_patterns.rst` file